precompiles/avs/types.go

```diff
@@ -68,38 +68,51 @@ func (p Precompile) GetAVSParamsFromInputs(_ sdk.Context, args []interface{}) (*
 		exoAddresses[i] = accAddr.String()
 	}
 	avsParams.AvsOwnerAddress = exoAddresses
-
+	// bech32
+	whitelistAddress, ok := args[7].([]string)
+	if !ok || whitelistAddress == nil {
```

Comment:
add verfication  **len(whitelistAddress) == 0**

Reply:
The whitelist can be empty so that any operator can optin this avs